c7n/resources/efs.py

```diff
@@ -193,3 +194,45 @@ def process(self, resources):
                     LifecyclePolicies=op_map.get(self.data.get('state')))
             except client.exceptions.FileSystemNotFound:
                 continue
+
+
+@ElasticFileSystem.filter_registry.register('lifecycle-policy')
+class LifecyclePolicy(Filter):
+    """Filters efs based on the state of lifecycle policies
+
+    :example:
+
+      .. code-block:: yaml
+
+            policies:
+              - name: efs-filter-lifecycle
+                resource: efs
+                filters:
+                  - type: lifecycle-policy
+                    state: present
+                    value: AFTER_7_DAYS
+
+    """
+    schema = type_schema(
+        'lifecycle-policy',
+        state={'enum': ['present', 'absent']},
+        value={'type': 'string'},
+        required=['state', 'value'])
```

Comment:
I think its valid to look for state: present or state: absent sans specifying a value here.

Reply:
Thanks, addressed this